Makefile

```diff
@@ -34,6 +34,7 @@ $(UNZIP_DIR)/Makefile:
 configure: $(UNZIP_DIR)/configure
 	# run configure
 	@cd $(UNZIP_DIR)  && \
+                ./bootstrap.sh && \
```

Comment:
Happy to do this. We are actually using this version internally in our group.

But this the ./bootstrap.sh needed?